jobs/sql/extractions/hiv_monthly_reporting.sql

```diff
@@ -96,6 +105,56 @@ set latest_hiv_viral_load_coded	= obs_from_group_id_value_coded_list(latest_hiv_
 update temp_eom_appts t
 set latest_hiv_viral_load	= obs_from_group_id_value_numeric(latest_hiv_viral_load_obs_group,'CIEL','856');
 
+-- latest arv treatment line 
+-- a temp table is loaded with the most recent HIV order information for each row
+-- this is used to update the main temp table
+drop temporary table if exists temp_last_arv_order_id;
+create temporary table temp_last_arv_order_id 
+	select t.monthly_reporting_id , o.order_id, o.date_activated, o.encounter_id  
+	from temp_eom_appts t 
+	inner join 
+		(select o2.order_id, o2.patient_id ,  o2.date_activated, o2.encounter_id  
+		from orders o2 
+		where o2.voided = 0
+		and o2.order_reason = concept_from_mapping('CIEL','138405')
+		order by if(o2.date_stopped is null,0,1) asc, o2.date_activated desc) o on o.patient_id = t.patient_id and o.date_activated < t.reporting_date  
```

Comment:
Not sure date_activated is reliable enough.  You might also need to look at scheduled_date, as the start date of the order can theoretically not be right at the activated date.  There are other reports/functions in which you have done this, so I'd refer to those.

Also, per above, this will only give you one (of possibly several) orders that are at the same time, for multiple meds